src/services/convert.service.ts

```diff
@@ -0,0 +1,49 @@
+import { convert } from '@asyncapi/converter';
+import { AsyncAPIDocument } from '@asyncapi/parser';
+
+import YAML from 'js-yaml';
+import { LAST_SPEC_VERSION } from '../interfaces';
+
+import { logger } from '../utils/logger';
+
+/**
+ * Service providing `@asyncapi/converter` functionality.
+ */
+export class ConvertService {
+  /**
+   * Convert the given spec to the desired language.
+   * @param spec AsyncAPI spec
+   * @param language Language to convert to, YAML or JSON
+   * @param version [version] AsyncAPI spec version
+   * @returns converted spec
+   */
+  public async convertSpec(
+    spec: AsyncAPIDocument | string,
+    language = '',
+    version: string = LAST_SPEC_VERSION,
+  ): Promise<string> {
+    try {
+      const asyncapiSpec = typeof spec === 'object' ? JSON.stringify(spec) : spec;
+      const convertedSpec = convert(asyncapiSpec, version);
```

Comment:
When we will merge that PR https://github.com/asyncapi/converter-js/pull/96 (we have bug in the `converter-js`, sorry!) we will be able to do only one operation:

```js
const convertedSpec = convert(spec, version);
```

rest will be handled inside `converter-js` package :)